### DISCUSSION TITLE
SEB and Kaltura-Videos

### DISCUSSION THREAD

**Discussion by moehzhaw** [SafeExamBrowser/seb-win-refactoring#828] [Q&A]
As I can see, there are already a lot of Q&A on using videos with SEB. (From a didactic point of view, using videos in an exam can be really important.) In the other Q&A, the answer to "Why does it not work?" is: because of codec.

Now, just to be sure: We have the same codec-story when using ebedded Kaltura videos. And this for Windows and macOS. Right?

**Comment by dbuechel** [comment]
I can only reply with respect to the Windows version: Yes, SEB 3.x does not support proprietary audio and video codecs, thus the issue persists no matter which web application or LMS you're using. As far as I know, the macOS and iOS versions do support proprietary codecs, but only @danschlet knows the exact details.

**Comment by moehzhaw** [comment]
Thanks. 
As we have BYOD, we'd have to run it on Windows and macOS... So, I do not need more informations at the moment.
